src/components/login/login.scss

```diff
@@ -1,25 +1,116 @@
 @import '../shared/styles/common.scss';
 @import '../shared/styles/flexbox';
 
+$formWidth: 225px;
+
 .loginView {
+    display: block;
+    position: fixed;
+    left: 0;
+    top: 0;
+    z-index: 9999;
     width: 100%;
     height: 100%;
+    background-color: rgba(0, 0, 0, 0.4);
 
-    .modalButtonBar {
-        display: flex;
-        justify-content: space-between;
-        padding-top: 10px;
+    .backgroundImage {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        z-index: -1;
     }
 
-    .label {
+    .wrapper {
+        position: fixed;
+        top: 50%;
+        left: 50%;
+        padding: 30px 90px;
+        transform: translate(-50%, -50%);
+        box-shadow: 0 0 0 1px rgba(0, 0, 0, .1), 0 3px 14px rgba(0, 0, 0, .4);
+        border-radius: 2px;
+        text-align: center;
+        overflow-y: auto;
         display: flex;
+        justify-content: center;
         flex-direction: column;
-        padding-top: 10px;
-    }
+        align-items: center;
+        color: #FFF;
+        background-color: #007ac9a6;
+
+        .header {
+            padding: 10px 0px 10px 0px;
+            user-select: none;
+
+            h2 {
+                margin-top: 10px 0px 10px 0px;
+            }
+        }
+
+        .form {
+            display: flex;
+            flex-direction: column;
+            width: $formWidth;
+            font-size: $mediumFontSize;
+
+            input {
+                height: 20px;
+                margin: 10px 0px 10px 0px;
+                padding: 15px;
+                border: 1px solid #FFF;
+                background: transparent;
+                color: #FFF;
+                border-radius: 2px;
+            }
+
+            ::placeholder {
+                color: #FFF;
+            }
+
+            .checkboxContainer {
+                display: flex;
+                align-items: center;
+                cursor: pointer;
+                user-select: none;
+
+                .checkbox {
+                    position: relative;
+                    top: 2px;
+                    width: 15px;
+                    margin: 0px 10px 0px 0px;
+                }
+            }
+        }
+
+        .button {
+            display: flex;
+            flex-basis: 40px;
+            justify-content: center;
+            flex-direction: column;
+            user-select: none;
+            width: $formWidth;
+            position: relative;
+            margin-top: 20px;
+            cursor: pointer;
+            font-weight: 400;
+            white-space: nowrap;
+            font-size: $mediumFontSize;
+            border: hidden;
+            border-radius: 2px;
+            background-color: #ffffffe6;
+            color: $fontColorDefault;
```

Comment:
Quite many of these properties are not needed: at least on my machine both: position, font-weight, white-space and border has these properties by default. except for border, but not needed to specify the border as hidden, when we have not added any border for it.

Reply:
Some css taken out